harvester/utils.py

```diff
@@ -83,6 +83,30 @@ def construct_mets_download_location(
     return Path(base_path) / Path(file_dir) / Path(filename)
 
 
+def construct_dir_structure(
```

Comment:
Except on a second (third?) thought, this would indeed benefit from slight renaming anyway: the other two functions make it clear which path they are actually constructing, but just "directory structure" will be absolutely undecipherable in couple of months. Whether we keep the verb or not, we could do something like `(construct_)binding_location` or something like that.

Reply:
I removed verbs from all path-making functions in `utils` to keep things uniform.